coriolisclient/cli/providers.py

```diff
@@ -76,3 +91,23 @@ def take_action(self, args):
         pvs = self.app.client_manager.coriolis.providers
         obj_list = pvs.list().providers_list
         return ProvidersFormatter().list_objects(obj_list)
+
+
+class ListProviderSchemas(lister.Lister):
+    """ List provider schemas by provider type """
+
+    def get_parser(self, prog_name):
+        parser = super(ListProviderSchemas, self).get_parser(prog_name)
+        parser.add_argument('platform', help='The platform\'s name')
+        parser.add_argument(
+            'type',
+            help="Provider type. Accepted values: "
+                 "['connection', 'destination', 'source']")
```

Comment:
Please try to avoid hardcoding stuff in helpstrings. Just format in `list(PROVIDER_SCHEMA_TYPE_MAP.keys())`